src/api/app/models/event/added_user_to_group.rb

```diff
@@ -0,0 +1,29 @@
+module Event
+  class AddedUserToGroup < Base
+    self.description = 'Added user to group'
+    payload_keys :group, :user, :who
+
+    receiver_roles :user
+
+    def subject
+      "You were added to the group '#{payload['group']}'" unless payload['who']
+
+      "'#{payload['who']}' added you to the group '#{payload['group']}'"
+    end
+
+    def users
+      [User.find_by(login: payload['user'])]
+    end
+
+    def originator
+      payload_address('who')
+    end
+
+    def parameters_for_notification
+      group_id = Group.find_by(title: payload['group']).id
+
+      super.merge({ notifiable_type: 'GroupsUser',
```

Comment:
Help me understand, why is the notifiable a `GroupUser` (the in-between model) and not `Group`?

Reply:
Semantically, we notify about a new association between a Group and a User, that's GroupsUser.

In fact, I started using Group as notifiable, but I then thought it made more sense like this.

Reply:
Semantically we notify about a change to a `Group`.

Reply:
I see. I'll change it.

Reply:
Done!